lib/pbench/server/api/resources/intake_base.py

```diff
@@ -87,7 +119,16 @@ def process_metadata(metas: list[str]) -> JSONOBJECT:
 
         We allow the client to set metadata on the new dataset. We won't do
         anything about this until upload is successful, but we process and
-        validate it here so we can fail early.
+        validate it first so we can fail early.
+
+        This supports typed metadata expressions using the syntax key : value
+        : type, such as `global.key:1:int`, `global.key:true:bool`, or
+        `global.key:'{"a": true, "b": 1}`:json`.
```

Comment:
You have a back-quote after the closing brace where I think you want a single-quote.